File: Java/M01JavaProgrammingBasics/L01FirstStepsInCoding/Lab/E07ProjectsCreation.java
package bg.softuni.programming_basics.first_steps_in_coding.lab;

import java.util.Scanner;

public class E07ProjectsCreation {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String nameArchitecture = scanner.nextLine();
        int projectsNum = Integer.parseInt(scanner.nextLine());

        int projectCreation = 3;

        int result = projectCreation * projectsNum;

        System.out.printf("The architect %s will need %d hours to complete %d project/s.", nameArchitecture, result, projectsNum);
    }
}

File: C++/M01_ProgrammingBasics/L01_FirstStepsInCoding/Lab/E07ProjectsCreation.cpp
#include <iostream>
using namespace std;

int main()
{
    string name;
    cin >> name;

    int numOfProjects;
    cin >> numOfProjects;

    int timeNeedForASingleProject = 3;

    int totalTime = timeNeedForASingleProject * numOfProjects;

    cout << "The architect " << name << " will need " << totalTime << " hours to complete " << numOfProjects << " prject/s.";
    return 0;
}
